indico/modules/events/registration/controllers/management/regforms.py

```diff
@@ -316,6 +316,18 @@ def _process(self):
         return redirect(url_for('.manage_regform_list', self.event))
 
 
+class RHRegistrationFormClone(RHManageRegFormBase):
+    """Clone a registration form."""
+
+    def _process(self):
+        form = clone_registration_form(self.regform)
+        flash(_('Registration form cloned. You can now modify it.'), 'success')
+        logger.info('Registration form %s cloned by %s', self.regform, session.user)
```

Comment:
We could also include the new form here as well:
```suggestion
        logger.info('Registration form %r cloned into %r by %r', self.regform, form, session.user)
```